src/events.js

```diff
@@ -607,16 +607,26 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
 };
 
 /**
- * Checks if the participant has a clinical blood or urine collected variable under baseline. If participant has clinical blood or urine collected, show a notification and return true.
+ * Checks if the participant has any specimen collected (clinical blood or clinical urine) under baseline from the dashboard or collected clinical blood or urine derivations from the site EMR API. If participant has either a clinical blood or urine dashboard variable or other blood/urine derived variables from the site EMR API, show a notification and return true.
  * @param {Object} data - participant data
- * @returns {Boolean} - true if participant has any clinical blood or urine collected, false otherwise
+ * @returns {Boolean} - true if participant has any clinical blood or urine collected derivations, false otherwise
 */
 const checkClinicalBloodOrUrineCollected = (data) => {
```

Comment:
Maybe we could call this variable `participantData` instead of `data` just to benefit anyone scanning the code since a lot of this code has biospecimen data?

Reply:
Thanks for the suggestion. I'll make the change.